### ISSUE TITLE
Bug  while adding dependencies  

### ISSUE THREAD

**Issue by vinitpoojary** [burhanrashid52/PhotoEditor#110]
has i add dependencies  ` implementation 'ja.burhanrashid52:photoeditor:0.3.3'`

I get error

`Could not resolve all dependencies for configuration ':app:debugRuntimeClasspath'.
Could not load module metadata from C:\Users\Vinit\.gradle\caches\modules-2\metadata-2.53\descriptors\com.android.support\support-v13\27.1.1\6740fe48b44caed96d7f50b66ff7b0b5\descriptor.bin
Unexpected metadata type found.
`

**Comment by burhanrashid52** [comment]
Can you provide some more details? Hope you have added jcenter and maven

**Comment by vinitpoojary** [comment]
why am i getting class path error any idea

**Comment by burhanrashid52** [comment]
Did you try clearing gradle cache ?

**Comment by vinitpoojary** [comment]
ya i tried that and  i even uninstall and reinstall android studio didn't work for me

**Comment by vinitpoojary** [comment]
yes still getting error i tried in my friends laptop its work  fine over there every thing is same in both of our laptop but still gets this descriptor.bin
what does this error means
```
Could not resolve all dependencies for configuration ':app:debugRuntimeClasspath'.
Could not load module metadata from C:\Users\Vinit\.gradle\caches\modules-2\metadata-2.53\descriptors\com.android.support\support-v13\27.1.1\6740fe48b44caed96d7f50b66ff7b0b5\descriptor.bin
Unexpected metadata type found.

```

**Comment by burhanrashid52** [comment]
Gradle is not able to load the desciptor.bin file. Try deleting .gradle/cache folder

**Comment by vinitpoojary** [comment]
same problem

**Comment by vinitpoojary** [comment]
hi just solved the problem just deleted the distributed.bin from C:\Users\Vinit\.gradle\caches\modules-2\metadata-2.53\descriptors\com.android.support\support-v13\27.1.1\6740fe48b44caed96d7f50b66ff7b0b5\descriptor.bin
thanks for all the help

**Comment by burhanrashid52** [comment]
You can close the issue if its resolved
